scripts/setup/restore-backup

```diff
@@ -93,9 +98,8 @@ def restore_backup(tarball_file):
                 "zulip",
             ]
         )
-        as_postgres = ["su", "-s", "/usr/bin/env", "-", "--", POSTGRES_USER]
-        run(as_postgres + ["dropdb", "--if-exists", "--", db_name])
-        run(as_postgres + ["createdb", "-O", "zulip", "-T", "template0", "--", db_name])
+        run(["dropdb", "--if-exists", "--", db_name])
+        run(["createdb", "-O", "zulip", "-T", "template0", "--", db_name])
```

Comment:
So, this ends up changing this block of code to run as root rather than the postgres user, which isn't great for security; why do we need to do that?  If the issue is passing environment variables, we just need to modify the `as_postgres`  command to pass them as e.g. `PGDATABASE=x` as part of the `as_postgres` line.

Reply:
@timabbott The reason I did was that because there is no `postgres` user present in docker zulip container. I guess the `postgres` user is created only if the database is present. Does it makes sense to run those commands as `zulip` user instead from a security persepctive? Or should we be doing something else.

Reply:
Yes, I think if there is no postgres user, we should just have  `as_postgres` use the zulip user instead (or equivalently, set `POSTGRES_USER='zulip'`).  We should avoid running any code as root that doesn't have to be.